charge/servers/molecular_property_utils.py

```diff
@@ -129,6 +129,8 @@ def calculate_property_hf(smiles: str, property: PropertyType) -> float:
 
     Returns
     -------
+    property : str
```

Comment:
argument, not return type
```suggestion
    str
```